ansible_runner/utils.py

```diff
@@ -239,13 +239,16 @@ def write(self, data):
             if stdout_actual and stdout_actual != "{}":
                 if not self.suppress_ansible_output:
                     sys.stdout.write(stdout_actual + "\n")
+                    self._handle.flush()
```

Comment:
Why `self._handle.flush()` here and not `sys.stdout.flush()`? It does not seem to be working.

Reply:
good point